sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/WebExceptionRetryPolicy.java

```diff
@@ -3,28 +3,32 @@
 
 package com.azure.cosmos.implementation;
 
+import com.azure.cosmos.CosmosException;
 import com.azure.cosmos.implementation.apachecommons.lang.time.StopWatch;
 import com.azure.cosmos.implementation.directconnectivity.WebExceptionUtility;
+import com.azure.cosmos.implementation.http.HttpTimeoutPolicy;
+import com.azure.cosmos.implementation.http.HttpTimeoutPolicyDefault;
+import com.azure.cosmos.implementation.http.ResponseTimeoutAndDelays;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import reactor.core.publisher.Mono;
 
+import java.net.URI;
 import java.time.Duration;
 import java.util.concurrent.TimeUnit;
```

Comment:
Not introduced by this PR - this unused import can be removed.

Reply:
Done